modules/oadp-release-notes-1-1-2.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-1-2_{context}"]
+= OADP 1.1.2 release notes
+
+The OADP 1.1.2 release notes include product recommendations.
+
+== Product recommendations
+
+Before you install OADP 1.1.2, it is recommended to either install VolSync 0.5.1 or to upgrade to it.
```

Comment:
Volsync 0.6.0 is released, there might be change that user will directly install 0.6.0 instead of 0.5.1.

Reply:
Should I include what you wrote above in the Product recommendation?